src/ja/server/diagrams/cancelLowLevel.puml

```diff
@@ -0,0 +1,55 @@
+
+@startuml
+
+box "Central Server" #Lightblue
+    participant ":CancelActivity"
+    participant ":Scheduler"
+    participant ":Authenticator"
+    participant ":DataBaseHandler"
+    participant ":Dispatcher"   
+end box
+
+
+-> ":CancelActivity":
+activate ":CancelActivity"
+
+":CancelActivity" -> ":Scheduler" ++: cancelJob(jobID:int, userID:int)
+
+note right of ":CancelActivity"
+  user calls 
+  "cancel" command
+end note
+
+":Scheduler" -> ":Authenticator" ++:checkPermission(jobID, userID)
+":Authenticator" --> ":Scheduler" --: isPermitted:bool
+
+alt isPermitted
+    ":Scheduler" -> ":DataBaseHandler" ++: checkQueue(jobID)
+    ":DataBaseHandler" --> ":Scheduler" --:isQueued:bool
+
+    break isQueued
+        ":Scheduler" -> ":DataBaseHandler" ++: removeFromQueue(jobID)
+        ":DataBaseHandler" --> ":Scheduler" --: 
+        ":Scheduler" -> ":Dispatcher" ++:  removeFromQueue(jobID)
+        ":Dispatcher" --> ":Scheduler" --:
+        
+        ":Scheduler" --> ":CancelActivity" : message
+    else
+
+    ":Scheduler" -> ":DataBaseHandler" ++: removeRunningJob(jobID)
+    ":DataBaseHandler" --> ":Scheduler" --
+    ":Scheduler" -> ":Dispatcher" ++: removeRunningJob(jobID)
+    ":Dispatcher" -> ":Dispatcher" ++: findJob(jobID) 
+    ":Dispatcher" --> ":Dispatcher" --: wmID:int
+
+    ":Dispatcher" -> ":Dispatcher" ++: stop(jobID, wmID)
+    ":Dispatcher" --> ":Dispatcher" --: 
+    ":Dispatcher" --> ":Scheduler" --: message
+    ":Scheduler" --> ":CancelActivity" --: message
+    <-- ":CancelActivity" : message
```

Comment:
Cancelling a Job that is already running should work like this:
* Retrieve Job from Database
* Tell the Dispatcher to kill the Job
* The Dispatcher get the id of the WorkerClient from the Job object.
* The Dispatcher instructs a WorkerClientProxy to kill the Job.
* If successful, update Database and re-evaluate which Jobs should be run on which WorkerClients. This re-evaluation is the **exact same** like when a Job is added by a user.